experimental/chromium-bitcode-embedding.patch

```diff
@@ -0,0 +1,105 @@
+diff --git a/AUTHORS b/AUTHORS
+index 4b62a980e90c9..ac00eb5c5c859 100644
+--- a/AUTHORS
```

Comment:
do we need the change to AUTHORS - since this is likely used a s an on the fly patch, at most it may cause patching problems

Reply:
Good point. Definitely something unnecessary that could potentially hurt maintainability. I've removed it from the patch.